builtin/credential/ldap/path_config.go

```diff
@@ -106,6 +106,11 @@ Default: cn`,
 				Default:     "tls12",
 				Description: "Maximum TLS version to use. Accepted values are 'tls10', 'tls11' or 'tls12'. Defaults to 'tls12'",
 			},
+			"deny_null_bind": &framework.FieldSchema{
+				Type:		 framework.TypeBool,
+				Default:	 false,
```

Comment:
I'm fine with minor breaks so long as there is a security story around it, and I think there is here. That said, a roughly equivalent version is to just set the default of deny_null_bind to true, which won't break any existing setups, so maybe that's the better option.